backend/src/db/migrations/20230909170408_change_meditation_entries.ts

```diff
@@ -0,0 +1,40 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+  MEDITATION_ENTRIES: 'meditation_entries',
+  MEDITATION_TOPICS: 'meditation_topics',
+};
+
+const ColumnName = {
+  ID: 'id',
+  TOPIC_ID: 'topic_id',
+  AUDIO_URL: 'audio_url',
+} as const;
+
+type MeditationTopic = {
+  id: number;
+};
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TableName.MEDITATION_ENTRIES, (table) => {
+    table.integer(ColumnName.TOPIC_ID).nullable().alter();
+    table.text(ColumnName.AUDIO_URL).notNullable().defaultTo('');
+  });
+}
+
+async function down(knex: Knex): Promise<void> {
+  const topic = await knex(TableName.MEDITATION_TOPICS)
+    .select<MeditationTopic>(ColumnName.ID)
+    .first();
+
+  await knex.schema.alterTable(TableName.MEDITATION_ENTRIES, (table) => {
```

Comment:
```suggestion
  return knex.schema.alterTable(TableName.MEDITATION_ENTRIES, (table) => {
```

Reply:
In async functions I can't return unawaited promise:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/d5f1d89e-2451-4d30-b625-4099508345b4)
Also I can't return awaited promise:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/6f628ef0-2400-4cae-9570-16296dc2d353)
At now it's the only way to handle it)

Reply:
ok got it